poke-line.el

```diff
@@ -96,24 +78,55 @@ Minimum of 3 units are required for poke-line."
   :set (lambda (sym val)
          (set-default sym val)
          (poke-line-refresh))
-  :group 'poke)
+  :group 'poke-line)
+
+(defun poke-line-refresh ()
+  "Refresh poke-line.
+Intended to be called when customizations were changed, to reapply them immediately."
+  (force-mode-line-update))
+
+(defun poke-line-set-pokemon (name)
+  "Choose active pokemon by NAME."
+  (interactive
+   (list (completing-read "Who do you choose? "
+                          (mapcar #'car poke-line-pokemon-types) nil 'require-match)))
+  (let* ((pokemon (assoc-string name poke-line-pokemon-types)))
+    (unless pokemon (user-error "Couldn't find Pokemon \"%s\"" name))
+    (setq poke-line-pokemon name)
+    (poke-line-refresh)
+    (message "%s, i choose you!" (upcase-initials name))))
+
+(defun poke-line-set-random-pokemon ()
+  "Choose a Pokemon at random."
+  (interactive)
+  (pcase-let*
+      ((i (random (length poke-line-pokemon-types)))
+       (`(,name . ,type) (elt poke-line-pokemon-types i)))
```

Comment:
Minor, but can you change i to a more descriptive variable name?

Reply:
Done